lib/salus/config.rb

```diff
@@ -35,7 +35,8 @@ class Config
     ).freeze
 
     DEFAULT_SCANNER_CONFIG = {
-      'pass_on_raise' => false # strong default - if a scanner raises, it counts as failure.
+      'pass_on_raise' => false, # strong default - if a scanner raises, it counts as failure.
+      'scanner-timeout-ms' => 0 # Scanner can run indefinitely if a max lifespan isn't provided
```

Comment:
We don't really need the timeout here.
If timeout is not found in config, we'll just assume there is no timeout and run everything without the timeout stuff.

Reply:
This does what you've mentioned. A timeout of 0 means that a scanner can run indefinitely (based on [the timeout docs](https://ruby-doc.org/stdlib-2.5.1/libdoc/timeout/rdoc/Timeout.html)).